source/dem/visualization.cc

```diff
@@ -94,6 +94,7 @@ Visualization<dim>::print_xyz(
     }
 
   pcout << "id, type, dp, x, y, z " << std::endl;
```

Comment:
why is this the only I/o with pcout? Maybe this one should be also with std::cout instead. Just manually done with the proc 0.

Reply:
I checked it with std::cout, as you said. But again proc1 calls the print_xyz function first in insert_list_2d test and the header is written at the end of the output file.

Reply:
Hmm this is weird. Might be a CI-only related issue.